man/systemd.network.xml

```diff
@@ -2572,6 +2572,18 @@ NFTSet=prefix:netdev:filter:eth_ipv4_prefix</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>BOOTP=</varname></term>
+        <listitem>
+          <para>Takes a boolean. The DHCPv4 client can be configured to communicate with BOOTP servers that
+          don't accept Option 53, DHCP Message Type. In this configuration, a BOOTP Request is sent without
+          any options by default instead of trying DHCU. A BOOTP reply that contains Option 1: Subnet Mask is
```

Comment:
> A BOOTP reply that contains Option 1: Subnet Mask is expected.

Is it verified?

Reply:
I cannot follow. networkd guesses what?? Which part of documentation? Could you elaborate more?

Reply:
@evverx Thank you for the reference.

Yeah, let's drop the check and the documentation about subnet mask.